apps/design/frontend/src/clone_election_button.tsx

```diff
@@ -0,0 +1,140 @@
+import { assert } from '@votingworks/basics';
+import { Election } from '@votingworks/types';
+import { P, Button, Modal, ButtonVariant, Icons, Font } from '@votingworks/ui';
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import * as api from './api';
+import { OrgSelect } from './org_select';
+import { useUserFeatures } from './features_context';
+
+export interface CloneElectionButtonProps {
+  election: Election;
+  variant?: ButtonVariant;
+}
+
+const OrgModal = styled(Modal)`
+  /* Allow modal to grow with user zoom setting and cap near screen height. */
+  min-height: min(40rem, 98%);
+`;
+
+const Tooltip = styled.span`
+  /*
+   * [TODO] No easy way to use theme colors for transparency, since they're
+   * defined in 'HSL' - need to bring in the 'polished' lib used in libs/ui, or
+   * create a generalized tooltip component in libs/ui.
+   */
+  background: rgba(0, 0, 0, 75%);
+  border-radius: 0.25rem;
+  bottom: calc(100% + 0.7rem);
+  box-shadow: 0.1rem 0.1rem 0.2rem 0.05rem rgba(0, 0, 0, 25%);
+  color: #fff;
+  font-weight: ${(p) => p.theme.sizes.fontWeight.semiBold};
+  padding: 0.5rem 0.75rem 0.6rem;
+  position: absolute;
+  right: 0;
+  width: max-content;
+
+  &:hover {
+    /* Prevent it from sticking around when moving quickly between buttons. */
+    display: none !important;
+  }
+`;
+
+const ButtonContainer = styled.span`
+  position: relative;
+
+  ${Tooltip} {
+    display: none;
+  }
+
+  &:focus,
+  &:hover {
+    ${Tooltip} {
+      display: block;
+    }
+  }
+`;
+
+export function CloneElectionButton(
+  props: CloneElectionButtonProps
+): React.ReactNode {
+  const { election, variant } = props;
+
+  const history = useHistory();
+  const features = useUserFeatures();
+  const user = api.getUser.useQuery().data;
+
+  const [orgId, setOrgId] = React.useState<string | undefined>(user?.orgId);
+  const [modalActive, setModalActive] = React.useState(false);
+
+  const cloneMutation = api.cloneElection.useMutation();
+  const mutateCloneElection = cloneMutation.mutate;
```

Comment:
Just curious - why not inline in L76 as `cloneMutation.mutate(...`?

Reply:
Ah, mostly working around an overly eager eslint check - would ideally put `cloneMutation.mutate` in the dependency list and then use it directly in the callback, but that triggers an eslint requiring `cloneMutation` to be in the deps list